### DISCUSSION TITLE
Datepicker in modal window

### DISCUSSION THREAD

**Discussion by mahyuddin** [themesberg/flowbite-svelte#209] [Q&A]
As Datepicker import datepicker javascript library in svelte header, we set rel='external' in inside a link component. But, how about if we put Datepicker inside modal window? How to pass rel='external'?  Or is there any other approach , so we can use Datepicker in modal window?

**Comment by shinokada** [comment]
I haven't tested it yet. But this should work.
Let's say you have a modal on modal-page. 
You need to add  `rel='external'` to the link to the page:

```
<a href="modal-page"  rel='external'>Modal page</a>
```

**Comment by AlekseiBuzlov** [comment]
Same issue - datepicker doesn't work in modal window
